src/main/java/duke/util/Parser.java

```diff
@@ -0,0 +1,158 @@
+package duke.util;
+
+import duke.Duke;
+import duke.commands.*;
```

Comment:
I think you should explicitly state the imports as stated [here.](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements)